test/common/http/http1/codec_impl_test.cc

```diff
@@ -908,6 +908,67 @@ TEST_P(Http1ServerConnectionImplTest, Http10MultipleResponses) {
   }
 }
 
+// SPELLCHECKER(off)
+struct {
+  const absl::string_view http_version;
+  const absl::optional<absl::string_view> balsa_parser_expected_error;
+  const absl::optional<absl::string_view> http_parser_expected_error;
+} kRequestHTTPStringTestCases[] = {{"", {}, {}}, // HTTP/0.9 has no HTTP-version.
+                                   {"HTTP/9.1", {}, {}},
```

Comment:
Consider having some known versions in here? e.g. 1.0, 1.1? so any breakage in the code would catch those

Reply:
Good idea.  Done.